app/lib/forms/choose_your_npq.rb

```diff
@@ -41,6 +50,8 @@ def next_step
         end
       elsif course.ehco?
         :about_ehco
+      elsif approved_itt_provider? && lead_mentor_course?
+        :possible_funding
```

Comment:
Does this mean that this always leads to be a user being funded?

Reply:
If so, we need to adjust the Services::FundingEligibility to update the user's funding status based on this information. Once that's done the next if statement, ` if eligible_for_funding?` would navigate the user accordingly.

Reply:
If we don't update Services::FundingEligibility then we'll be directing the user down the "you are expected to be funded" path without adjusting how the eligible_for_funding or funding_eligiblity_status_code data stored against their application is calculated.

Reply:
As i understand yes? although also the symbol :possible_funding seems a bit misleading! I guess the provider still has to accept the application

Reply:
Provider acceptance is unrelated and happens after funding eligibility is calculated. 

There's an element of backwards logic going on here. You go to possible_funding because your eligible_for_funding value was true. That's what the next line is doing. Rather what we have here is doing that data check then sending the user on without adjusting how funding eligibility is actually calculated.

If a user selecting an approved ITTProvider and having a lead mentor course should change the result of eligible_for_funding when the application is saved and sent to the ECF DB then you need to update Services::FundingEligibility to correctly calculate their funding, as it stands the above change only updates what information the user is shown and doesn't change the result of their application.